TopNews/app/src/main/java/com/fisfam/topnews/adapter/HomeAdapter.java

```diff
@@ -40,6 +41,7 @@
 
     public HomeAdapter(final Context context, final RecyclerView recyclerView) {
         super();
+        mItems.add(0,"List of categories");
```

Comment:
add kiểu gì si đa một cái string thế này, tạo hẳn object mới CategoryList sth...

Reply:
No chi la 1 cai Object bat ki nao do de luc getViewType nhan ra thoi. Tao ra 1 class moi chi de cho moi the thoi?

Reply:
Khi vd tôi muốn kéo xuống 10 articles, hiển thị 1 cái list category nữa, thì Long lại hardcoded cái vị trí tiếp? khi cái danh sách có 100 cái articles, thì long phải hardcoded 9 lần nữa?

Trong trường hợp này thì đương nhiên không cần thiết lắm, nhưng phải nghĩ linh hoạt để sau này maintain dễ hơn. Không nên hardcoded bất cứ cái gì nếu có thể làm khác.